runWebpack.sh

```diff
@@ -167,6 +167,7 @@ if [[ "$NODE_ENV" == "dev" || "$NODE_ENV" == "auto" ]]; then
   echo ./node_modules/.bin/webpack-dev-server
   ./node_modules/.bin/webpack-dev-server
 else
+  export NODE_OPTIONS='--max-old-space-size=4096'
```

Comment:
is this needed?

Reply:
https://github.com/uc-cdis/data-portal/pull/674#discussion_r412477816 😆

Reply:
proof of this person doesn't memorize things 🤦